engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -120,6 +122,27 @@ class SpelExpression(parsed: ParsedSpelExpression,
   }
 }
 
+object NullExpression {
```

Comment:
Can u extract it to separate class? IMO it is not specially spel related. We can use it with any other expression language...

Reply:
Moved to `expression` package